dhall/CHANGELOG.md

```diff
@@ -1,3 +1,21 @@
+1.37.2
+
+* [BUG FIX: Fix CORS compliance check](https://github.com/dhall-lang/dhall-haskell/pull/2121)
+    * Previous versions were not correctly enforcing CORS compliance
+    * This implies that some imports that would have worked previously by
+      accident will now fail; specifically: an import from one domain
+      transitively importing something from another domain that has not opted
+      into CORS
+* [Detect preferred character set from input](https://github.com/dhall-lang/dhall-haskell/pull/2108)
```

Comment:
The `Expr` changes look like they should entail a major version bump. `dhall-{docs,json,lsp-server,nix,openapi}` were also affected and need new releases.

Reply:
I'll create a new pull request for 1.38.0, then (otherwise there is a good chance that I forgot to fix the version number when I merge).